code/game/jobs/job/antag/xeno/queen.dm

```diff
@@ -10,7 +10,7 @@
 	return spawn_positions
 
 /datum/job/antag/xenos/queen/transform_to_xeno(mob/new_player/NP, hive_index)
```

Comment:
Arguments should follow parent's implementation:
![image](https://github.com/cmss13-devs/cmss13/assets/76988376/0876bbf6-4d22-4eb6-bb26-6edcfb61070c)

```suggestion
/datum/job/antag/xenos/queen/transform_to_xeno(mob/living/carbon/human/human_to_transform, hive_index)
```

Reply:
going to git line blame the guy before me that set that arg.